_integration-schemas/xy/v1/customer.md

```diff
@@ -0,0 +1,104 @@
+---
+tap: "x-y"
+version: "1.0"
+key: "customer"
+
+name: "customer"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-x-y/blob/master/tap_x_y/schemas/customer.json"
+description: |
+  The `{{ table.name }}` table contains information about customers in your {{ integration.display_name }} account.
+
+replication-method: "Key-based Incremental"
+
+api-method:
+    name: ""
+    doc-link: ""
```

Comment:
I know XY doesn't have public docs, so it's fine that this isn't filled out. When this isn't being used, please comment it out. That way it won't create an empty field when Jekyll builds the site.